ciphers/autoclave.py

```diff
@@ -0,0 +1,111 @@
+def encrypt(plaintext, key) -> str:
+    """
+    Function that encrypt a given plaintext (string)
+    and key (string), returning the encrypted ciphertext
+
+    @params
+    plaintext - a normal text to be encrypted (string)
+    key - a small text or word to start the replacing (sFtring)
+
+    @return
+    A string with the ciphertext
+
```

Comment:
```suggestion
```
Typically, we want the parameter names to be self-documenting so this isn't really necessary